CONTRIBUTING.md

```diff
@@ -127,11 +127,11 @@ If you're happy to run tests only against an SQLite database, you can skip this
 
 If you have Docker installed, use any of the following commands to start fresh local databases of the dialect of your choice:
 
-- `yarn start-mariadb`
-- `yarn start-mysql`
-- `yarn start-postgres`
+- `yarn start-mariadb-oldest` (for MariaDB 10.3.0) or `yarn start-mariadb-latest` (for MariaDB 10.9.3)
```

Comment:
```suggestion
- `yarn start-mariadb-oldest` (for MariaDB 10.3) or `yarn start-mariadb-latest` (for MariaDB 10.9)
```

Let's not fix this on patch versions for now so it requires less updates. We're still discussing internally which patch versions we support

Reply:
Fixed in latest commit